spec-tests/src/test/java/org/candlepin/spec/SubscriptionResourceSpecTest.java

```diff
@@ -100,7 +100,7 @@ public void shouldSubscriptionsDerivedProvidedProductsReturnEmptyArrayWhenNull()
             .productId(prod.getId())
             .quantity(2L)
             .subscriptionId(StringUtil.random("test"))
-            .subscriptionSubKey("master");
+            .subscriptionSubKey("primary");
```

Comment:
value change -- omitting comment on other instances within this file

Reply:
done